### PR TITLE
Time DTS without seconds

### PR DESCRIPTION
**Pull request by LegitDongo**
## Description
Time DTS without seconds
In the case of hours being available, it will show hours as well as minutes
Includes documentation update

This was a frequently asked for feature

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Motivation and Context
Frequently requested.
Some people don't like the seconds on the end
It also makes PA compatible with other tools

## How Has This Been Tested?
Locally, Windows 10

## Wiki Update
- [x] This change requires an update to the Wiki.
- [ ] This change does not require an update to the Wiki.

### CODE REVIEW

File: PokeAlarm/Utils.py [hunk 1]
```diff
@@ -481,7 +481,15 @@ def get_time_as_str(t, timezone=None):
...
     time_24 = disappear_time.strftime("%H:%M:%S")
-    return time_left, time_12, time_24
+
+    # Get the same as above but without seconds
+    time_left_minutes = "%dm" % m if h == 0 else "%dh %dm" % (h, m)
+    time_12_minutes = disappear_time.strftime("%I:%M") \
+        + disappear_time.strftime("%p").lower()
+    time_24_minutes = disappear_time.strftime("%H:%M")
+
+    return time_left, time_12, time_24, \
```
Comment (kvangent) [inline PokeAlarm/Utils.py]:
```suggestion
    return time, time_12h, time_24h, \
```

File: PokeAlarm/Utils.py [hunk 2]
```diff
@@ -481,7 +481,15 @@ def get_time_as_str(t, timezone=None):
...
+        time_left_minutes, time_12_minutes, time_24_minutes
```
Comment (kvangent) [inline PokeAlarm/Utils.py]: v
```suggestion
        time_no_sec, time_12h_no_sec, time_24h_no_sec
```

File: docs/configuration/events/egg-events.rst [hunk 3]
```diff
@@ -75,16 +75,22 @@ geofence            Geofence around the event.
...
+======================= ===============================================================
+DTS                     Description
+======================= ===============================================================
+hatch_time_left         Time remaining until the egg hatches.
+12h_hatch_time          Time when the egg will hatch, formatted in 12h.
+24h_hatch_time          Time when the egg will hatch, formatted in 24h.
+raid_time_left          Time remaining until the raid ends.
+12h_raid_end            Time when the raid ends, formatted in 12h.
+24h_raid_end            Time when the raid ends, formatted in 24h.
+hatch_time_left_minutes Time remaining until the egg hatches without seconds.
```
Comment (kvangent) [inline docs/configuration/events/egg-events.rst]: Maybe we should update these more explicitly? (formatted h?:mm:ss) vs (formatting h?:mm)

File: PokeAlarm/Events/EggEvent.py [hunk 4]
```diff
@@ -79,6 +79,14 @@ def generate_dts(self, locale, timezone, units):
             '12h_raid_end': raid_end_time[1],
             '24h_raid_end': raid_end_time[2],
 
+            # Time Remaining Without Seconds
```
Comment (kvangent) [inline PokeAlarm/Events/EggEvent.py]: Separating on seconds/no_seconds seems weird, maybe separate on hatch_time, raid_end_time instead?

File: PokeAlarm/Events/EggEvent.py [hunk 5]
```diff
@@ -79,6 +79,14 @@ def generate_dts(self, locale, timezone, units):
...
+            'hatch_time_left_minutes': hatch_time[3],
```
Comment (kvangent) [inline PokeAlarm/Events/EggEvent.py]: I don't like minutes, because it implies that the other doesn't have minutes. How about `hatch_time_no_secs`?

**Review by kvangent — CHANGES_REQUESTED** [review]

**Comment by kvangent** [comment]
@LegitDongo Can we address feedback and get this merged?

File: docs/configuration/events/egg-events.rst (hunk 3 above)
Comment (LegitDongo) [reply docs/configuration/events/egg-events.rst]: I'm thinking of adding an example column so that it's defined a little more explicitly

**Review by kvangent — DISMISSED** [review]

**Review by kvangent — APPROVED** [review]

**Pull request merged**